index.js

```diff
@@ -0,0 +1,280 @@
+      "population": 9531712,
+      "region": "Asia",
+      "capital": "Amman"
+  },
+  {
+      "name": "Kazakhstan",
+      "flag": "https://flagcdn.com/w320/kz.png",
+      "population": 17753200,
+      "region": "Asia",
+      "capital": "Astana"
+  },
+  {
+      "name": "Kenya",
+      "flag": "https://flagcdn.com/w320/ke.png",
+      "population": 47251000,
+      "region": "Africa",
+      "capital": "Nairobi"
+  },
+  {
+      "name": "Kiribati",
+      "flag": "https://flagcdn.com/w320/ki.png",
+      "population": 113400,
+      "region": "Oceania",
+      "capital": "South Tarawa"
+  },
+  {
+      "name": "Kuwait",
+      "flag": "https://flagcdn.com/w320/kw.png",
+      "population": 4183658,
+      "region": "Asia",
+      "capital": "Kuwait City"
+  },
+  {
+      "name": "Kyrgyzstan",
+      "flag": "https://flagcdn.com/w320/kg.png",
+      "population": 6047800,
+      "region": "Asia",
+      "capital": "Bishkek"
+  },
+  {
+      "name": "Laos",
+      "flag": "https://flagcdn.com/w320/la.png",
+      "population": 6492400,
+      "region": "Asia",
+      "capital": "Vientiane"
+  },
+  {
+      "name": "Latvia",
+      "flag": "https://flagcdn.com/w320/lv.png",
+      "population": 1961600,
+      "region": "Europe",
+      "capital": "Riga"
+  },
+  {
+      "name": "Lebanon",
+      "flag": "https://flagcdn.com/w320/lb.png",
+      "population": 5988000,
+      "region": "Asia",
+      "capital": "Beirut"
+  },
+  {
+      "name": "Lesotho",
+      "flag": "https://flagcdn.com/w320/ls.png",
+      "population": 1894194,
+      "region": "Africa",
+      "capital": "Maseru"
+  },
+  {
+      "name": "Liberia",
+      "flag": "https://flagcdn.com/w320/lr.png",
+      "population": 4615000,
+      "region": "Africa",
+      "capital": "Monrovia"
+  },
+  {
+      "name": "Libya",
+      "flag": "https://flagcdn.com/w320/ly.png",
+      "population": 6385000,
+      "region": "Africa",
+      "capital": "Tripoli"
+  },
+  {
+      "name": "Liechtenstein",
+      "flag": "https://flagcdn.com/w320/li.png",
+      "population": 37623,
+      "region": "Europe",
+      "capital": "Vaduz"
+  },
+  {
+      "name": "Lithuania",
+      "flag": "https://flagcdn.com/w320/lt.png",
+      "population": 2872294,
+      "region": "Europe",
+      "capital": "Vilnius"
+  },
+  {
+      "name": "Luxembourg",
+      "flag": "https://flagcdn.com/w320/lu.png",
+      "population": 576200,
+      "region": "Europe",
+      "capital": "Luxembourg"
+  }
+]
+// console.log(CountriesData);
+// console.log("hi");
+// import countries from './CountriesData.json' 
+//console.log(CountriesData)
+let countriesGrid = document.querySelector(".countries-grid");
+const searchInput=document.querySelector("[data-search]")
+
+searchInput.addEventListener("input",e=>{
+  const value=e.target.value.toLowerCase()
+  //console.log(value)
+  const searchedCountries = CountriesData.filter(
+      (country) => country.name.toLowerCase().includes(value)
+    );
```

Comment:
In case the value is `""` do you want to filter everything? or maybe return the entire data as is?